roles/KubevirtCommonTemplatesBundle/tasks/main.yml

```diff
@@ -16,135 +16,26 @@
   set_fact:
     deployed_templates_after: "{{ lookup('k8s', api_version=ct_status.results[0].result.apiVersion, kind='template') }}"
 
-- name: "Get common-templates-cr"
```

Comment:
I want this change, and it looks good at glance. But it's hard to see how the old, convoluted code (because of k8s_status bugs) translatest to the new lean and mean one. Thus, please
- add comments/or amend the commit message to explain in a couple lines how the fixes in the k8s_status allow to simplify the logic (e.g. why it was convoluted before?)
- explicitely highlight if the new conditions refactored are 1:1 simplifications of the old logic or if you dropped some conditions, as it is hinted in the current commit message. E.g: which conditions where reported before? what now?